pkg/util/helper/binding.go

```diff
@@ -125,21 +121,29 @@ func (a *Dispenser) TakeByWeight(w ClusterWeightInfoList) {
 }
 
 // GetStaticWeightInfoListByTargetClusters constructs a weight list by target cluster slice.
-func GetStaticWeightInfoListByTargetClusters(tcs []workv1alpha2.TargetCluster) ClusterWeightInfoList {
-	weightList := make(ClusterWeightInfoList, 0, len(tcs))
-	for _, result := range tcs {
+func GetStaticWeightInfoListByTargetClusters(available, scheduled []workv1alpha2.TargetCluster) ClusterWeightInfoList {
```

Comment:
My suggestion is not to rename the parameters, as `tcs` is used everywhere, if you think it is worth renaming it, do it by a separate PR and add some comments on it.

Reply:
done